webui/cypress/integration/features/items.feature

```diff
@@ -3,22 +3,23 @@ Feature: Item Management
 
   Scenario: Create Item
     Given I logged in
-    When I go to "items" page
+    When I go to items page
     Then I should see page title "Items"
     And I should see some entries
     When I click "Create" button
     Then I should see page title "New Item"
     When I fill in "name" with "test item 1"
     And I fill in "description" with "description of test item"
     And I click "Submit" button
+    And wait for "@itemCreate"
```

Comment:
Including `@itemCreate` here breaks the pattern of writing these feature specs in "plain English". Would it be too much trouble to structure this in a way that it could say something more readable like, `And wait for item creation`?

Reply:
it would mean add another 6 `When('wait for xxx')` definition. I don't think we need to follow the "plain English" pattern as a dogma. I don't think it hurt readability a lot here, does it?

Reply:
I don't think it's dogmatic to ask if there's way to write this as plain English 😕

If it's not too big of imposition, I think it would be more readable to use plain English whenever possible. Sounds like in this case it would be a serious imposition, though.